kibble/api/languages.go

```diff
@@ -64,6 +71,51 @@ func loadAllLanguagesFromApi(cfg *models.Config, site *models.Site) error {
 	return nil
 }
 
+// Returns an error if:
+// - default language is not set.
+// - default language is missing display name.
+// - no site languages are set.
+// - any site language is missing a display name.
+// - default language is not included in supported languages.
+func (l languagesV1) validate() error {
+	isZero := func(val string) bool {
```

Comment:
Nit: semantics, zero implies a number type to me.
```suggestion
	isEmpty := func(val string) bool {
```

Reply:
Potentially could have made it a variadic function for later usage being `if isEmpty(code, label, name)` but only mentioning out of interest.

Reply:
Zero values are a thing in Golang... empty values aren't.

https://go.dev/tour/basics/12
https://pkg.go.dev/reflect#Value.IsZero

Reply:
TIL